### ISSUE TITLE
./databox up fails when -v is not set

### ISSUE THREAD

**Issue by ivyleavedtoadflax** [ukgovdatascience/databox#17]
Seems to be a problem parsing the default arguments...

```
>>> ./databox.sh -i c4.4xlarge up
Creating DataBox
1 error(s) occurred:

* aws_ebs_volume.volume: size: cannot parse '' as int: strconv.ParseInt: parsing "": invalid syntax
The state file either has no outputs defined, or all the defined
outputs are empty. Please define an output in your configuration
with the `output` keyword and run `terraform refresh` for it to
become available. If you are using interpolation, please verify
the interpolated value is not empty. You can use the
`terraform console` command to assist.
```

**Comment by ivyleavedtoadflax** [comment]
https://www.terraform.io/docs/configuration/interpolation.html#conditionals

**Comment by ivyleavedtoadflax** [comment]
resolved in #19
